code/modules/clothing/gloves/color.dm

```diff
@@ -191,6 +180,19 @@
 	max_heat_protection_temperature = GLOVES_MAX_TEMP_PROTECT
 	strip_delay = 60
 	armor = list(MELEE = 0, BULLET = 0, LASER = 0, ENERGY = 0, BOMB = 0, BIO = 0, RAD = 0, FIRE = 70, ACID = 50)
+	resistance_flags = NONE
+
+/obj/item/clothing/gloves/color/chief_engineer
+	desc = "These gloves provide excellent heat and electric insulation. They are so thin you can barely feel them."
```

Comment:
```suggestion
	desc = "These gloves provide excellent heat and electric insulation. They are so thin that you can barely feel them."
```
someone might need to check me grammar-wise on this one

Reply:
its ok